### ISSUE TITLE
Routing doesn't seem to work properly

### ISSUE THREAD

**Issue by ParadauxIO** [JohnSloan8/abair#1]
Hi John,

Noticed that whatever you're using for routing seems to work cosmetically but not functionally.
Take Geabaire for example. If I click on Geabaire from
`Applications -> Accessibility -> Geabaire` it will correctly redirect me to the page for that project, and update the URL bar to say `https://abair.ie/geabaire` however, if I try and link directly to `https://abair.ie/geabaire`  elsewhere on the internet (for example via my Github Profile) it will not work, it'll give me the Apache 404 page (which we should also introduce a custom 404 page)  

Try clicking on this link without having the SPA open to reproduce 
https://abair.ie/geabaire

**Comment by JohnSloan8** [comment]
Yep, it works fine on Cloudflare (click https://abair.pages.dev/geabaire), but we decided to host on the Abair server. After realising the SPA routing doesn't work, there wasn't time to fix before launch. All urls with the abair.ie/ domain were setup to redirect to the homepage though, so I'll check what is going on with the Apache 404 page. Tbh, might be best just to point 'abair.ie' at the Cloudflare site, rather than spend time configuring Apache routing.

**Comment by ParadauxIO** [comment]
I'd agree with just pointing @ at cloudflare. Cloudflare supports CNAME flattening so it should work without any additional effort from you.

You can also configure CF pages to build multiple branches so you could create a production branch which is built separately to your master/main.

You'll need to get a signed DPA with cloudflare in order to use it as a service provider although i think that's what I'm going to do for the AAC for similar reasons (and have aac.abair.ie resolve to that)
